packages/frontend/src/components/accounts/recovery_setup/new_account/SetupPassphraseNewAccount.js

```diff
@@ -0,0 +1,76 @@
+import { KeyPair } from 'near-api-js';
+import { generateSeedPhrase } from 'near-seed-phrase';
+import React, { useEffect, useState } from 'react';
+
+import { Mixpanel } from '../../../../mixpanel/index';
+import ConfirmPassphrase from './ConfirmPassphrase';
+import SavePassphrase from './SavePassphrase';
+
+export default ({ handleConfirmPassphrase }) => {
+    const [passPhrase, setPassPhrase] = useState('');
+    const [recoveryKeyPair, setRecoveryKeyPair] = useState();
+    const [implicitAccountId, setImplicitAccountId] = useState('');
+
+    const [confirmPassphrase, setConfirmPassphrase] = useState(false);
+    const [wordId, setWordId] = useState(null);
```

Comment:
1. :nit: `wordIndex` would be a bit clearer (`id` usually means something like a UUID)
2. Rather than track `wordId`, why not track the word itself -- e.g. `verificationWord` or something like that? It would make the component easier to follow and eliminate the entire concept of tracking the index just to look up the word from this component, since we never use the index for anything else